tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -57,7 +65,7 @@ def Listen(self, callback):
     else:
       print("No service account. Using application default credentials")
       # pylint: disable=unused-variable
-      credentials, project_id = auth.default()
```

Comment:
this is deleting the correct one: https://google-auth.readthedocs.io/en/master/reference/google.auth.html

Reply:
Thanks for catching that. Fixed.